README.md

```diff
@@ -1 +1,31 @@
-# Buidler-Starter
\ No newline at end of file
+# Buidler-Starter
+
+## Requirements
+
+Following software is required to be installed to use this repo:
+* [NodeJS](https://nodejs.org/en/) >= v12.0.0
+* [Nomic Labs buidler](https://buidler.dev/getting-started/#overview)
```

Comment:
isn't this and deploy part of the dependencies, so they don't have to be installed separately?

Reply:
Yes, but wanted to mention that those two are used, will change `readme` to something more appropriate 🙂